BrainPortal/app/views/tasks/_params.html.erb

```diff
@@ -44,11 +44,36 @@
   :
 </h2>
 
-<div class="generalbox">
-<%= error_messages_for(@task) %>
-<% if raw_partial.(:task_params) %>
-  <%= render :inline => raw_partial.(:task_params), :locals => locals %>
-<% else %>
-  <%= render :partial => task_partial('task_params'), :locals => locals %>
+<%
+input_notes =
+  @task.class.superclass == PortalTask ?
```

Comment:
This is really not right. The current view file, `_params.html.erb`, does not and should not know anything about the internals of the new Boutiques integrator. It is not correct for it to invoke the method `descriptor_for_before_form` because that is not a method of the core CbrainTask class.

What you have to do is add a PortalTask method called, maybe "input_notes" that by default returns `[]` with a comment that says subclasses can provide an array of short helpful comments. And in the BoutiquesPortalTask class, you override it.